src/Microsoft.AspNetCore.Sockets.Abstractions/DefaultConnectionContext.cs

```diff
@@ -22,12 +22,12 @@ public class DefaultConnectionContext : ConnectionContext,
                                             IConnectionHeartbeatFeature,
                                             ITransferFormatFeature
     {
-        private object _heartbeatLock = new object();
+        private readonly object _heartbeatLock = new object();
```

Comment:
Resharper strikes again :). I'll have you know you reminded me it's time for my quasi-annual "Install the Resharper Trial and see if it's fast enough (and compatible with VsVim) to use" event. Things are looking better than they have in the past.